config/executor/config.yaml

```diff
@@ -2,6 +2,8 @@ application:
   clusterId : "Cluster1"
   pool: "default"
   submitThreadCount: 1
```

Comment:
just curious why you would have different numbers of threads for different operations?  Is it that we don't want to ddos the k8s api and some operations are more expensive than others.

Also re the naming- is this really a thread count here?  It seems more like a concurrency limit?

Reply:
Possibly its just a concurrency limit, as its effectively:
 - How many go routines do we use to submit/update/delete that can talk to the kube-api concurrently

Happy to change the values here.

I wanted delete to be > submit, just so we don't end up clogging the cluster up as we can submit things faster than delete them.

Also we perform operations in approximately this ratio:
 - 1 submit
 - 1 delete
 - 5 updates
Per job.

So have more update "threads" is to account for this. 
 - It could reasonably set to 5